oauthproxy.go

```diff
@@ -41,13 +41,15 @@ const (
 	schemeHTTPS     = "https"
 	applicationJSON = "application/json"
 
-	robotsPath        = "/robots.txt"
-	signInPath        = "/sign_in"
-	signOutPath       = "/sign_out"
-	oauthStartPath    = "/start"
-	oauthCallbackPath = "/callback"
-	authOnlyPath      = "/auth"
-	userInfoPath      = "/userinfo"
+	robotsPath         = "/robots.txt"
+	bulmaCssPath       = "/css/bulma/bulma.min.css"
+	fontAwesomeCssPath = "/css/font-awesome/all.min.css"
```

Comment:
I would expect the CSS files to be registered under the proxy prefix otherwise the paths could interfere with upstream servers, wdyt?

Also, typically these kind of assets come under a static path, how about `<proxy-prefix>/static/...`?